### DISCUSSION TITLE
2. Wechselrichter Phasenabhänig steuern

### DISCUSSION THREAD

**Discussion by rieders** [hoylabs/OpenDTU-OnBattery#395] [General]
Hallo
Ich habe opendtu Battery mit einem Wechselrichter am laufen.
Als Power-Meter habe ich ein SDM630 per MQTT eingebunden, das geht soweit auch.
Nun möchte ich das der Wechselrichter 1 sich um die 1. Phase "kümmert" und der Wechselrichter 2 sich um die Regelung der 2. Phase bemüht.
Ist das mit opendtu Battery möglich?

Noch eine 2. Frage
Ich habe den VE.Direkt Anschluss mit an  opendtu Battery angelötet. Kann ich damit auch an Generic Venus Device (rpi or other) oder kann ich nur an den Victron Laderegler?

Grüße
Andre

**Comment by bf4ed** [comment]
Baue einfach einen zweiten DTU. Ich hab drei. Sind doch nicht so teuer.
DTU1 für HM-1500 solar für regelung auf 600W
DTU2 für HM-800 auf Batterie und für's laden mit Huawei
DTU3 für HM-600 auf Batterie

**Reply by rieders** [reply]
Das stimmt natürlich. Ich dachte es könnte eine openDTU regeln.

grüße

**Reply by schlimmchen** [reply]
> Das stimmt natürlich. Ich dachte es könnte eine openDTU regeln.

Das wird sicherlich irgendwann mal möglich sein, viele Leute haben Interesse daran, aber da muss noch einiges an Hirnschmalz und Arbeit investiert werden. Siehe #230
